src/Land/Model/soil_water.jl

```diff
@@ -14,19 +14,21 @@ The defaults are no moisture anywhere, for all time.
 # Fields
 $(DocStringExtensions.FIELDS)
 """
-struct PrescribedWaterModel{F1, F2} <: AbstractWaterModel
+struct PrescribedWaterModel{FN1, FN2} <: AbstractWaterModel
     "Augmented liquid fraction"
-    ϑ_l::F1
+    ϑ_l::FN1
     "Volumetric fraction of ice"
-    θ_ice::F2
+    θ_ice::FN2
```

Comment:
we also need to change all the _ice to _i in this file

Reply:
nvm, was looking at an old version!

Reply:
weird it was changed on my end ( i did a control shift F search) on climatemachine and replaced all θ_ice with θ_i, not sure why this is there